src/_includes/install/sampledata/sample-data-clone.md

```diff
@@ -87,6 +87,11 @@ To clone the Magento sample data GitHub repository using the SSH protocol:
 
 1. See [Set file system permissions and ownership](#samp-data-perms).
 
+1. Run the following command:
+   ```bash
```

Comment:
Add an empty line before ```bash

Thank you!

Reply:
@atwixfirster I added an empty line.
Please proceed.
Thank you!

Reply:
Looks ok to me. Thank you, @DashukVolodymyr 👍 

@rogyar , could you please review this PR as a maintaner.

Thank you!